### ISSUE TITLE
RNA-seq: Reached anomalous state during search for most likely exon alignment

### ISSUE THREAD

**Issue by komalsrathi** [Illumina/strelka#52]
Hi

I am running into some errors but unsure on figure out how to fix it. Here is my command for one RNA-seq sample:

```
$strelka-2.9.4.release_src/bin/configureStrelkaGermlineWorkflow.py --bam=sample.bam --referenceFasta=ucsc.hg19.fa --rna --runDir=$outdir

$outdir/runWorkflow.py -m local -j 8
```

Here are my log files for this run:
[strelka_run_logs.zip](https://github.com/Illumina/strelka/files/2124488/strelka_run_logs.zip)

Please let me know if you need more information from my side.

Thanks
Komal

**Comment by sangtaekim** [comment]
Can you send me the data?

**Comment by komalsrathi** [comment]
Hi - you can find the files here: 

Input bam: https://s3.amazonaws.com/strelka-test/sample.bam
Reference fasta: https://s3.amazonaws.com/strelka-test/ucsc.hg19.fa

Thank you!

**Comment by sangtaekim** [comment]
Can you change the permissions? I cannot access the files.

**Comment by komalsrathi** [comment]
That's weird - this bucket is public. Can you try again and let me know if you still can't access the files?

**Comment by sangtaekim** [comment]
It works now. Thanks!

**Comment by sangtaekim** [comment]
Sorry for the delay. I'm looking into this now.

**Comment by sangtaekim** [comment]
There was a recently introduced bug that caused this error. We've released v2.9.6 with a fix (https://github.com/Illumina/strelka/releases/tag/v2.9.6). Thanks @komalsrathi for the report!

**Comment by sangtaekim** [comment]
I confirmed that v2.9.6 works well on your data.
